eventorganizer.api/src/main/java/com/event/organizer/api/appuser/AppUserService.java

```diff
@@ -114,6 +118,20 @@ public AppUser changeAccountStatus(String currentUserEmail, AccountStatusRequest
         return userRepository.save(editedUser);
     }
 
+    public AppUser uploadProfilePicture(String currentUserEmail, String imageUrl) {
```

Comment:
This should be void instead, we don't need to return AppUser or anything at all as we only change a column value in the database here.